tests/acceptance/reminder-list-test.js

```diff
@@ -13,12 +13,12 @@ test('viewing the homepage', function(assert) {
   visit('/');
 
   andThen(function() {
-    assert.equal(currentURL(), '/');
+    assert.equal(currentURL(), '/reminders');
     assert.equal(Ember.$('.spec-reminder-item').length, 5);
```

Comment:
Looks like the word `Ember` here is unnecessary. Can you offer documentation for why it's needed?